pages/index.vue

```diff
@@ -58,6 +58,10 @@ div.landing-page {
     padding: $offset-4;
     text-align: center;
 
+    .link-primary {
+      width: 100%;
```

Comment:
this is nitpicking, but I would just add `.w-100` onto the `NuxtLink.link-primary` component